fs/ggml/ggml.go

```diff
@@ -378,10 +377,6 @@ func DetectContentType(b []byte) string {
 // maxArraySize. If maxArraySize is 0, the default value of 1024 is used. If
```

Comment:
I think the comment needs to be updated now that 0 isn't a special value anymore